src/storage/v2/indices.hpp

```diff
@@ -131,6 +131,10 @@ class LabelIndex {
   Config::Items config_;
 };
 
+struct IndexStats {
+  double stat_value, avg_group_size;
```

Comment:
what is stat_value?
maybe better naming

Reply:
> what is stat_value?

Statistical value. I will change it to the statistic